testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/AbstractJavascriptTest.java

```diff
@@ -1,4 +1,4 @@
-package org.keycloak.testsuite.adapter.javascript;
+package org.keycloak.testsuite.javascript;
```

Comment:
@vramik Shouldn't we use Assume.assumeTrue(AUTH_SERVER_SSL_REQUIRED); also for JavascriptTests?

Reply:
@mhajas it depends if we're going to fix the [issue](https://issues.jboss.org/browse/KEYCLOAK-9912) or not :) And it's not ma call, @hmlnarik wdyt?